SalesforceCommerceCloud/pwa-kit
# Combine API config files

# Description



Combine `commerce-api.config.js` and `einstein-api.config.js` into a single file:`api.config.js`

# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# Changes

- update import paths
- update generator

# How to Test-Drive This PR

- `node packages/pwa-kit-create-app/scripts/create-mobify-app-dev.js --outputDir /tmp/project`
- `cd /tmp/project`
- `npm start`
- verify app works with products and einstein recommendations

# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
